04-onion-routing.md

```diff
@@ -998,11 +1000,13 @@ The amount in the HTLC doesn't match the value in the onion.
 
 1. type: UPDATE|20 (`channel_disabled`)
 2. data:
-   * [`u16`:`flags`]
+   * [`u16`:`disabled_flags`]
    * [`u16`:`len`]
    * [`len*byte`:`channel_update`]
 
 The channel from the processing node has been disabled.
+No flags for `disabled_flags` are currently defined, thus it is currently
```

Comment:
In eclair this is actually the `channel_update.message_flags` byte followed by the `channel_update.channel_flags` byte, so it won't be `0x0000`. It is redundant with the contents of the `channel_update` though, so I really don't know why a separate field was included in that error.

Reply:
Yea, honestly it seems like an oversight that there's extra bytes here, but given its otherwise redundant I see no reason to copy the channel update flags, better to just set them to 0 and move on.